python/catkin/builder.py

```diff
@@ -520,9 +530,11 @@ def build_cmake_package(
 
     # Run make
     if not use_ninja:
-        make_executable = 'make'
-    else:
         make_executable = 'ninja'
```

Comment:
Is this condition inverted? It looks like it now says:

```python
if not use_ninja:
    make_executable = 'ninja'
```

That would seem to be the inverse of what is actually the intent.